sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -563,8 +251,6 @@ OpGroupAsyncCopyLocalToGlobal(__spv::Scope Execution, dataT *Dest, dataT *Src,
 
 extern void __spirv_ocl_prefetch(const char *Ptr, size_t NumBytes) noexcept;
 
-#endif // !__SYCL_DEVICE_ONLY__
-
 extern SYCL_EXTERNAL void __spirv_ControlBarrier(__spv::Scope Execution,
```

Comment:
Will we emit the rest of SPIR-V built-ins definitions by clang?

NOTE: I have opened https://github.com/intel/llvm/pull/1373, which has conflicting changes.

Reply:
I'm not sure of what you mean, this is part is related to host

Reply:
Sorry, I didn't notice that this is host-only code now.